src/pages/edge/lifecycle-for-edge-network/metrics.md

```diff
@@ -1,17 +1,68 @@
+
+### Application
+
+| **Property** | **Type** | **Description** |
+| :--- | :--- | :--- |
+| xdm:id | String | Identifier of the application. |
+| xdm:name | String | Name of the application. |
+| xdm:version | String | Version of the application. |
+| xdm:isLaunch | boolean | Launch of an application. Every application foreground event sets `isLaunch` to `true`. |
+| xdm:isInstall | boolean | Install of an application. If `true`, signifies the first launch of the application. The Experience Event's timestamp property can be used as the application's install date. |
+| xdm:isUpgrade | boolean | Upgrade of an application. If `true`, signifies the first launch of the application after an upgrade. |
+| dc:language | String | The language of the application to represent the user's linguistic, geographical, or cultural preferences for data presentation. |
+
+### Device
+
+| **Property** | **Type** | **Description** |
+| :--- | :--- | :--- |
+| xdm:type | String | Type of device being tracked. |
+| xdm:manufacturer | String | The name of the organization who owns the design and creation of the device. |
+| xdm:model | String | The name of the model for the device. |
+| xdm:modelNumber | String | The unique model number designation assigned by the manufacturer for this device. |
+| xdm:screenHeight | integer | The number of vertical pixels of the device's active display in the default orientation. |
+| xdm:screenWidth | integer | The number of horizontal pixels of the device's active display in the default orientation. |
+
+### Environment
+
+| **Property** | **Type** | **Description** |
+| :--- | :--- | :--- |
+| xdm:type | String | The type of the application environment. |
+| xdm:carrier | String | A mobile network carrier or MNO, also known as a wireless service provider, wireless carrier, cellular company, or mobile network carrier. |
+| xdm:operatingSystem | String | The name of the operating system used when the observation was made. |
+| xdm:operatingSystemVersion | String | The full version identifier for the operating system used when the observation was made. |
+| dc:language | String | The language of the environment to represent the user's linguistic, geographical, or cultural preferences for data presentation. |
+
+## Lifecycle Application Background metrics and dimensions
+
+The following data is collected on each [Lifecycle Application Background](../../home/base/mobile-core/lifecycle/event-reference.md#lifecycle-application-background) event. The structure of this data is defined in the Experience Data Model (XDM) field group [AEP Mobile Lifecycle Details](https://github.com/adobe/xdm/blob/master/docs/reference/adobe/experience/aep-mobile-lifecycle-details.schema.md).
```

Comment:
Unfortunately the actual schema is named "AEP Mobile Lifecycle Details" and I'd like to keep that in the documentation in case someone was to search for the schema in the Data Collection UI or the XDM docs.
https://github.com/adobe/xdm/blob/master/docs/reference/adobe/experience/aep-mobile-lifecycle-details.schema.md